makefiles/tools/targets.inc.mk

```diff
@@ -4,6 +4,8 @@
 # (as `all`), so `all` will always be the first target defined and thereby the
 # default target when `make` is called without any further argument.
 
+.PHONY: rsmb
```

Comment:
I have the feeling that this target name is too generic and should maybe be `mqtt_rsmb` or `mosquitto_rsmb` instead.

Reply:
`mosquitto_rsmb` it is :-)